src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -86,6 +86,7 @@
 
     .bcs-activity-feed-comment-input {
         background-color: $white;
+        flex-shrink: 0;
```

Comment:
What are are the other two `flex` values here? does it inherit `flex-grow` and `flex-basis` from somewhere else already? Should we use the shorthand syntax to be more deliberate?

Reply:
Good point, let me fix it to be more clear